datastore/elastic_search/query.py

```diff
@@ -134,15 +153,23 @@ def get_dictionary_records(connection, index_name, doc_type, entity_name, word_l
     return results
 
 
-def delete_dictionary_records_by_word(connection, index_name, doc_type, entity_name, word_list=None, **kwargs):
+def delete_entity_data_by_values(connection, index_name, doc_type, entity_name, values=None, **kwargs):
     """
+    Fetches entity data from ES for the specific entity
+    Args:
+        connection: Elasticsearch client object
+        index_name: The name of the index
+        doc_type: The type of the documents that will be indexed
+        entity_name (str): name of the entity for which the data is to be deleted. If None => All records are cleared
```

Comment:
Undocumented argument `values`. The second sentence `If None ...` seems to be for `values` argument
`values (list of str, optional): ...`